resources/js/App/Components/Layout/Layout.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import "./layout.css"
+
+import Header from "../Header/Header.jsx";
+import LayoutLeft from "../LayoutLeft/LayoutLeft.jsx";
+import LayoutRight from "../LayoutRight/LayoutRight.jsx";
+
+
+export default class Layout extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      desktop: true,
+      width: window.innerWidth,
+      videoPageOn: false,
+    }
+  }
+
+  componentWillMount() {
+    if(this.state.width < 900) {
+      this.setState( { desktop: false } )
+    }
+  }
+
+  isVideoPageOn = () => {
+    if(this.state.videoPageOn === false) {
+      this.setState ( { videoPageOn: true } )
+    } else {
+      this.setState ( { videoPageOn: false } )
+    }
+  }
+
+  render() {
+
+    if(this.state.desktop) {
+
+      return(
+        <div>
+          <Header />
+          <div className="layout-sides">
+            <LayoutLeft />
+            <LayoutRight />
+          </div>
+        </div>
+      )
+    } else {
```

Comment:
You don't really need to wrap it into `else` block, since you're returning from function in the `if` block.

```
if (this.state.desktop) {
  return ...
}

if (this.state.videoPageOn === false) {
  return ...
}

return ...
```

If you switch to media queries, some of this states will not be needed, so perhaps just do that part first.